lambdas/reconcile-migrated-vcs/src/main/java/uk/gov/di/ipv/core/reconcilemigratedvcs/domain/Request.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.di.ipv.core.reconcilemigratedvcs.domain;
+
+import java.util.List;
+
+public record Request(
+        String batchId,
+        List<String> userIds,
```

Comment:
Have you discussed a plan for constructing this list with Taher?

If we end up deciding to do 100k users at a time (the size of a batch), it'll be ~5MB of JSON which is getting quite close to the lambda limit (6MB).

(If it's only a small %, then it may not be a concern)

Reply:
I believe the plan is to export a CSV of IDs from the success audit events. As yet not discussed if we're going to reconcile everything or a sample.
i've been running tests with 100000 identities provided in the json without issue - although I should probably ensure that the format of the ids is the same...

Reply:
Looks like if we're using the same format of user ID we breach the limit. Simpliest thing is to just chop it in two and run smaller lists in to it.

Reply:
We could actually just run it in parallel - it doesn't mutate anything so shouldn't be an issue.